packages/db/tables.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TYPE public.pin_status AS ENUM (
```

Comment:
In niftysave we end up with a [following enum](https://github.com/ipfs-shipyard/nft.storage/pull/407/files#diff-4996c80e080afc9e4bf0eb737f86b756a6cf5b536012a8d6607e49a36c8792f3R118-R123 ) quoting below:

```sql
CREATE TYPE pin_status AS ENUM (
    'PinError',
    'PinQueued',
    'Pinned',
    'Pinning'
);
```

I would suggest using same as above here since:

1. It [represents a subset used by web3.storage](https://github.com/web3-storage/web3-schema/blob/7331b863b77f3bc0a71d1810a2fd14e813cfee9c/schema.sql#L74-L101)
2. Adding more variants to enum should not require record updates, while it would if we would need to rename values.

Reply:
@hugomrdias are you ok with this? It's fine with me.

Reply:
fine with me

Reply:
Now tracked in https://github.com/ipfs-shipyard/nft.storage/issues/331